pkg/cmd/codespace/view.go

```diff
@@ -0,0 +1,143 @@
+package codespace
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/cli/cli/v2/internal/codespaces/api"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/cli/cli/v2/utils"
+	"github.com/spf13/cobra"
+)
+
+const (
+	minutesInDay = 1440
+)
+
+type viewOptions struct {
+	selector *CodespaceSelector
+	exporter cmdutil.Exporter
+}
+
+func newViewCmd(app *App) *cobra.Command {
+	opts := &viewOptions{}
+
+	viewCmd := &cobra.Command{
+		Use:   "view",
+		Short: "View details about a codespace",
+		Long: heredoc.Doc(`
+			View details about a codespace.
+
+			For more fine-grained details, you can add the "--json" flag to view the full list of fields available.
+
+			If this command doesn't provide enough information, you can use the "gh api" command to view the full JSON response:
+			$ gh api /user/codespaces/<codespace-name>
+		`),
+		Example: heredoc.Doc(`
+			# select a codespace from a list of all codespaces you own
+			$ gh cs view	
+
+			# view the details of a specific codespace
+			$ gh cs view -c <codespace-name>
```

Comment:
With the way codespace names are formatted, doing something like `username-laughing-meme-4qj7qpjrqxx2pr5` seemed like it might be more confusing than leaving the placeholder but if you would prefer the example value I can add it.

Reply:
Yeah I think it would be best to change the placeholders to something like `codespace-name-12345` something that is the similar/same format as a standard codespace name but clearly an example. This command has a pretty straight forward use so I don't anticipate this causing any confusion. If we receive feedback that it is we can iterate.